spec/controllers/votes_controller_spec.rb

```diff
@@ -0,0 +1,36 @@
+require 'rails_helper'
+
+RSpec.describe VotesController, :type => :controller do
+
+  describe '#create' do
+    #### User factory girl to generate these - refactor opportunity
+    let!(:user) { User.create!(username: 'hello', email: 'hello@hello.com', password: "hello") }
+    let!(:idea) { Idea.create!(title: 'hellohello', description: 'hellohellohello', creator: user) }
+    before { allow(controller).to receive(:current_user) { user } }
+    let!(:comment) { idea.comments.create(author: user, text: 'hellohellohello') }
+
+    it 'creates a new vote from Idea' do
+      expect do
+        post :create, {
+          votable_type: 'Idea', votable_id: idea.id
+        }
+     end.to change{Vote.count}.by 1
+    end
+
+
+    it 'creates a new vote from Comment' do
+      expect do
+        post :create, {
+          comment: {
+            votable_type: 'Comment', votable_id: comment.id, text: 'this is a great idea'
+          }
+        }
+     end.to change{Vote.count}.by 1
```

Comment:
We've said that 100% test coverage doesn't mean your code is well tested. Here sort of an example. If our post didn't actually use any of the data we gave it and just created a blank comment, we wouldn't have any tests here that caught it for us.

You're on a good path, though!